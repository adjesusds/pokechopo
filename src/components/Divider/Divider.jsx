// External Packages
import React from 'react'

// Internal Packages
import StyledDivider from './styled/Divider'

/**
 * Creates a new Divider component.
 * @class
 */
const Divider = () => <StyledDivider />

export default Divider
