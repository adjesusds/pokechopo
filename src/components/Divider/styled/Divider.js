// External Packages
import styled from 'styled-components'

// Internal Packages
import { dimensions } from 'config/palette'

/**
 * @constant {Element} style The style of the component.
 */
const style = styled.span`
  margin: ${dimensions.xxxSmall};
`

export default style
