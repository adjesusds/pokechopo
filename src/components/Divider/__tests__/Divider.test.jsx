// External Packages
import { shallow } from 'enzyme'
import React from 'react'

// Internal Packages
import Divider from '../Divider'

describe('<Divider />', () => {
  it('should render the component on valid props', () => {
    const component = shallow(<Divider />)

    expect(component).toMatchSnapshot()
  })
})
