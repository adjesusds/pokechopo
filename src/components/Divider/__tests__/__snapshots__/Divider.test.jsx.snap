// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should render the component on valid props 1`] = `<styled.span />`;
